src/openrct2/ride/coaster/ReverseFreefallCoaster.cpp

```diff
@@ -385,6 +385,29 @@ static void paint_reverse_freefall_rc_vertical(
     }
 }
 
+static void paint_reverse_freefall_rc_on_ride_photo(
```

Comment:
You're drawing an on-ride photo section with LIM boosters here, even though it doesn't actually accelerate the train. You will need to draw it with a non-booster track (use the station track for this).

Reply:
![RFC](https://user-images.githubusercontent.com/61885939/81478294-331dba00-91d1-11ea-9d43-a041f767a695.png)
Would this work? This is derived from the station track piece drawing code, except I have removed all references to the station platform drawing and station identification codes.

Reply:
(This has since been discussed on Discord.)